src/rdiff_backup/rpath.py

```diff
@@ -997,8 +1000,22 @@ def get_temp_rpath(self, sibling=False):
             if self.__class__._temp_file_index > 100000000:
                 log.Log("Warning: Resetting tempfile index", 2)
                 self.__class__._temp_file_index = 0
-            tf = self.append('rdiff-backup.tmp.{index:d}'.format(
-                             index=self.__class__._temp_file_index))
+
+            if tempfile.tempdir and (shutil.disk_usage(self.path).free == 0):
+                # here to catch the bytes string passed via cli opts
```

Comment:
Since recently, we don't save `tempfile.tempdir` as bytes anymore because the tempfile module struggles with it (see comment in Main.py, there is  a bug for this).

I'm hesitating if I should tell you to not do the conversion, at the risk to hit a bug once the issue is solved, so leave it as it is. But modify the comment to clarify the purpose.